crates/openvino/src/model.rs

```diff
@@ -107,4 +113,121 @@ impl CompiledModel {
         ))?;
         Ok(InferRequest::from_ptr(infer_request))
     }
+
+    /// Get the input size of the compiled model.
+    pub fn input_size(&self) -> Result<usize> {
+        let mut input_size: usize = 0;
+        try_unsafe!(ov_compiled_model_inputs_size(self.ptr, &mut input_size))?;
+        Ok(input_size)
+    }
+
+    /// Get the single input port of the compiled model,
+    /// which only support single input model.
```

Comment:
In debug, it would throw an assertion and terminate -
```
        OPENVINO_ASSERT(inputs.size() == 1,
                        "CompiledModel::input() must be called on a compiled model with exactly one parameter.");
        return inputs.at(0);
```
In release, it would just return first (0th) index input